e2e/tests/join_call.spec.ts

```diff
@@ -109,8 +112,9 @@ test.describe('join call', () => {
         await userBPage.page.locator('[data-testid=SendMessageButton]').click();
 
         await userAPage.locator('.post__header').locator('button.user-popover').last().click();
-        await expect(userAPage.locator('#user-profile-popover')).toBeVisible();
-        await expect(userAPage.locator('#user-profile-popover').locator('#startCallButton')).toBeDisabled();
+        await expect(userAPage.locator('div.user-profile-popover')).toBeVisible();
+        await expect(userAPage.locator('div.user-profile-popover').locator('#startCallButton')).toBeDisabled();
+        await userAPage.locator('button.closeButtonRelativePosition').click();
```

Comment:
Just a thought -- will this be robust if another button is added on the page? Could we limit the locator to the `div.user-profile-popover` to make it more future proof?

Reply:
@cpoile with the new changes for pop over.
The testing tool encounters some kinda overlay when attempting to interacting with other elements while the profile pop over is open. This `button.closeButtonRelativePosition` is just a step to close the pop over and continue with other steps.

Another way was to `foce` click on elements, but I avoided that. This is just with the test tool. In real time, this works fine.